packages/plugins/rubygem-foreman_resource_quota/rubygem-foreman_resource_quota.spec

```diff
@@ -0,0 +1,93 @@
+# template: foreman_plugin
+%global gem_name foreman_resource_quota
+%global plugin_name resource_quota
+%global foreman_min_version 3.5.0
```

Comment:
current foreman version is 3.10. should we bump it?

Reply:
This comes from the plugin source. Please update it there if needed.